core/sherlock/backends/whoosh_backend.py

```diff
@@ -59,12 +59,11 @@ def index_file(self, filepath, *args, **kwargs):
         contents = safe_read_file(filepath)
         if contents is None:
             return
-        path = unicode(filepath)
```

Comment:
This function is not compatible with py3. As far as I tested, it removal seems not to change things:
- Unicode in file names was not supported before (threw errors in sqlite3 using python 2.7.10).
- Search (and retrieval) of terms containing unicode literals still works.